inc/shortcodes/class-scribd.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Shortcake_Bakery\Shortcodes;
+
+class Scribd extends Shortcode {
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => esc_html__( 'Scribd', 'shortcake-bakery' ),
+			'listItemImage'  => 'TK',
+			'attrs'          => array(
+				array(
+					'label'        => esc_html__( 'URL', 'shortcake-bakery' ),
+					'attr'         => 'url',
+					'type'         => 'text',
+					'description'  => esc_html__( 'Full URL to the Scribd document', 'shortcake-bakery' ),
+				),
+			),
+		);
+	}
+
+	public static function reversal( $content ) {
+		if ( preg_match_all( '#<iframe class="scribd_iframe_embed" src=[\'\"]([^\'\"]+)[\'\"].+data-auto-height=[\'\"][^\'\"]+[\'\"] data-aspect-ratio=[\'\"][^\'\"]+[\'\"] scrolling=[\'\"][^\'\"]+[\'\"] id=[\'\"][^\'\"]+[\'\"] width=[\'\"][^\'\"]+[\'\"] height=[\'\"][^\'\"]+[\'\"] frameborder="0"></iframe>?#', $content, $matches ) ) {
```

Comment:
@danielbachhuber As far as I can tell. We could be less strict in the matching algorithm I suppose.